server/swagger.json

```diff
@@ -3684,39 +3684,141 @@
-          "type": "string"
+          "type": "string",
+          "enum": [
+            "single-stat",
+            "line",
+            "line-plus-single-stat",
+            "line-stacked",
+            "line-stepplot",
+            "bar",
+            "gauge",
+            "table"
+          ],
+          "default": "line"
+        },
+        "colors": {
+          "description": "Colors define encoding data into a visualization",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DashboardColor"
+          }
+        },
+        "legend": {
+          "description":
+            "Legend define encoding of the data into a cell's legend",
+          "type": "object",
+          "properties": {
+            "type": {
+              "description": "type is the style of the legend",
+              "type": "string",
+              "enum": ["static"]
+            },
+            "orientation": {
+              "description":
+                "orientation is the location of the legend with respect to the cell graph",
+              "type": "string",
+              "enum": ["top", "bottom", "left", "right"]
+            }
+          }
+        },
+        "tableOptions": {
+          "timeFormat": {
+            "description":
+              "timeFormat describes the display format for time values according to moment.js date formatting",
+            "type": "string"
+          },
+          "verticalTimeAxis": {
+            "description":
+              "verticalTimeAxis describes the orientation of the table by indicating whether the time axis will be displayed vertically",
+            "type": "boolean"
+          },
+          "sortBy": {
+            "description":
+              "sortBy contains the name of the series that is used for sorting the table",
+            "type": "object",
+            "$ref": "#/definitions/RenamableField"
+          },
+          "wrapping": {
+            "description":
+              "wrapping describes the text wrapping style to be used in table cells",
+            "type": "string",
+            "enum": ["truncate", "wrap", "single-line"]
+          },
+          "fieldNames": {
+            "description":
+              "fieldNames represent the fields retrieved by the query with customization options",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/RenamableField"
+            }
+          },
+          "fixFirstColumn": {
+            "description":
+              "fixFirstColumn indicates whether the first column of the table should be locked",
+            "type": "boolean"
+          }
+        },
+        "links": {
+          "type": "object",
+          "properties": {
+            "self": {
+              "type": "string",
+              "description": "Self link mapping to this resource",
+              "format": "url"
+            }
+          }
         }
       },
       "example": {
```

Comment:
could update the example for cell to include tableoptions and more of the other options instead of being so empty